web/core/components/core/image-picker-popover.tsx

```diff
@@ -90,7 +90,8 @@ export const ImagePickerPopover: React.FC<Props> = observer((props) => {
   const { getRootProps, getInputProps, isDragActive, fileRejections } = useDropzone({
     onDrop,
     accept: {
-      "image/*": [".png", ".jpg", ".jpeg", ".webp"],
+      "image/jpeg": [],
```

Comment:
Does it allow jpg extension?

Issue only talks about removing .svg, we are removing webp as-well?

Reply:
Yes, it allows jpg extension. I have included screenshot of allowed extensions in the description. All allowed extensions are shown inside the "Files".

Yes, I was requested earlier today to remove webp as well.